Dockerfile-dc

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.13-alpine
+
+RUN apk add -U --no-cache ca-certificates
+
+ADD . /app
+WORKDIR /app
+ENV GO111MODULE on
+RUN go get -u github.com/gobuffalo/packr/v2/packr2
+RUN packr2
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build
+
+FROM scratch
```

Comment:
I think we can use the alpine-based image here, as people might want to `docker exec` into it to debug or whatever. That way, they'll have a shell available!

Check out another docker image that uses an alpine-based linux and adds a user `ory`. I'd recommend doing something similar: https://github.com/ory/kratos/blob/master/.docker/Dockerfile

Reply:
Got rid of the multistage multi image Docker and using `golang:1.13-alpine`